src/components/Header/MenuButton/MenuButton.component.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Container, StyledButton, StyledSvg, buttonPressedStyle} from '../HeaderIconStyles';
+
+
+export const MenuButton = () => {
+  return <Container data-testid="menuButton">
+        <StyledButton whileTap={buttonPressedStyle}>
+            <StyledSvg height="32" viewBox="0 0 512 512" width="32" data-testid="menuIcon">
+                <path d="m464.883 64.267h-417.766c-25.98 0-47.117 21.136-47.117 47.149 0 25.98 21.137 47.117 47.117 47.117h417.766c25.98 0 47.117-21.137 47.117-47.117 0-26.013-21.137-47.149-47.117-47.149z"/>
+                <path d="m464.883 208.867h-417.766c-25.98 0-47.117 21.136-47.117 47.149 0 25.98 21.137 47.117 47.117 47.117h417.766c25.98 0 47.117-21.137 47.117-47.117 0-26.013-21.137-47.149-47.117-47.149z"/>
+                <path d="m464.883 353.467h-417.766c-25.98 0-47.117 21.137-47.117 47.149 0 25.98 21.137 47.117 47.117 47.117h417.766c25.98 0 47.117-21.137 47.117-47.117 0-26.012-21.137-47.149-47.117-47.149z"/>
+            </StyledSvg>
```

Comment:
If you have more SVGs you can move them to their own folder, this improves the readability of the code